frontend/src/DialectReportView.tsx

```diff
@@ -1,19 +1,19 @@
 import CasesSection from "./components/Cases/CasesSection";
-import BowtieInfoSection from "./components/BowtieInfo/BowtieInfoSection";
 import RunInfoSection from "./components/RunInfo/RunInfoSection";
 import SummarySection from "./components/Summary/SummarySection";
-import { useMemo } from "react";
+import React, { useMemo } from "react";
```

Comment:
React import is unused, right?

Reply:
Removed and thanks for your help.

Reply:
Did CI not fail with an unused import? If not can you open a bug, though it looked like tooling in JS for imports is .. not great.